ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -105,9 +95,40 @@ void RSkComponentImage::OnPaint(
         canvas->restore();
     drawBorder(canvas,frame,imageBorderMetrics,imageProps.backgroundColor);
   } else {
-/* Emitting Image Load failed Event*/
-        imageEventEmitter->onError();
-    }
+  /* Emitting Image Load failed Event*/
+    RNS_LOG_ERROR("Image not loaded");
```

Comment:
can you add source uri str in the error message.
This way it will help in debugging to know for which image uri , this failure is releated to .

Reply:
sure